install/ubuntu/install.ubuntu.sh

```
@@ -0,0 +1,105 @@
+      service nginx stop 2>/dev/null
+      fi
+
+      # Kill running NetAlertX server processes in this INSTALL_DIR
+      pkill -f "python.*${INSTALL_DIR}/server" 2>/dev/null
+
+      # Unmount only if mountpoints exist
+      mountpoint -q "$INSTALL_DIR/api" && umount "$INSTALL_DIR/api" 2>/dev/null
+      mountpoint -q "$INSTALL_DIR/front" && umount "$INSTALL_DIR/front" 2>/dev/null
+
+      # Remove all contents safely
+      rm -rf -- "$INSTALL_DIR"/* "$INSTALL_DIR"/.[!.]* "$INSTALL_DIR"/..?* 2>/dev/null
```

Shellcheck (0.10.0)

[warning] Use "${var:?}" to ensure this never expands to /* .

(SC2115)